.github/workflows/build-release-packages.yml

```diff
@@ -11,6 +11,7 @@ on:
         required: true
 
 permissions:
+  id-token: write
```

Comment:
Why do we need the write permissions here?

Reply:
This is to ensure that your workflow has the necessary permission to perform OIDC authentication with AWS.